packages/ui-bindings/Person/getPersonById.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Method to call the person(id) query on Viewer
+ *
+ * @package: REA app
+ * @author:  Nicholas Roth <Lou3797>
+ * @since:   2018-02-08
+ */
+
+import { connect } from "react-redux";
+import { gql, graphql, compose } from "react-apollo";
+
+import { AppState } from "@vflows/store/types.js";
+import { getActiveLoginToken } from "@vflows/store/selectors/auth.js";
+
+import { personInterface } from "./personInterface";
+
+const query = gql`
+query($token: String, $PersonId: Int) {
```

Comment:
Naming variables with a capital letter (PersonId) can lead to confusion with objects.